app/src/main/java/com/framgia/soundclound/util/BindingUtil.java

```diff
@@ -0,0 +1,60 @@
+package com.framgia.soundclound.util;
+
+import android.databinding.BindingAdapter;
+import android.graphics.PorterDuff;
+import android.support.design.widget.TabLayout;
+import android.support.v4.content.ContextCompat;
+import android.support.v4.view.PagerAdapter;
+import android.support.v4.view.ViewPager;
+
+import com.framgia.soundclound.screen.main.TabType;
+
+/**
+ * Created by Sony on 1/4/2018.
+ */
+
+public class BindingUtil {
+
+    @BindingAdapter({"pagerAdapter"})
+    public static void setViewPagerAdapter(ViewPager viewPager, PagerAdapter adapter) {
+        viewPager.setAdapter(adapter);
+    }
+
+    @BindingAdapter({"selectedColor", "unSelectedColor"})
+    public static void setupTabIndicatorListener(final TabLayout tabLayout, final int selectedColor,
+                                                 final int unSelectedColor) {
+            tabLayout.setOnTabSelectedListener(new TabLayout.OnTabSelectedListener() {
+                @Override
+                public void onTabSelected(TabLayout.Tab tab) {
+                    if (tab.getIcon() == null) {
+                        return;
+                    }
+                    tab.getIcon().setColorFilter(ContextCompat.getColor(tabLayout.getContext(),
+                            selectedColor), PorterDuff.Mode.SRC_ATOP);
+                }
+
+                @Override
+                public void onTabUnselected(TabLayout.Tab tab) {
+                    if (tab.getIcon() == null) {
+                       return;
```

Comment:
ERROR: 'if' child have incorrect indentation level 23, expected level should be one of the following: 24, 28.